common/lib/client/clientoptions.js

```diff
@@ -0,0 +1,9 @@
+var ClientOptions = (function() {
+
+  /* public constructor */
+	function ClientOptions() {
```

Comment:
What's with the indentation here?  Don't you have a linter running?

Reply:
@mattheworiordan Not yet. Related issue here: https://github.com/ably/ably-js/issues/441